tests/System.IO.Compression.Tests/BrotliStreamTests.cs

```diff
@@ -67,13 +67,18 @@ public void CanDisposeBaseStream()
         public void CanDisposeBrotliStream()
         {
             var ms = new MemoryStream();
-            var bro = new BrotliStream(ms, CompressionMode.Compress);
-            bro.Dispose();
-
-            // Base Stream should be null after dispose
-            Assert.Null(bro._stream);
-
-            bro.Dispose(); // Should be a no-op
+            using (var bro = new BrotliStream(ms, CompressionMode.Compress))
```

Comment:
even for internals and locals, we try to use the naming conventions. In this case, I would just call it stream or compressionStream, not "bro"